README.md

```diff
@@ -72,7 +72,7 @@ policy may be made to more closely align with other providers in the Cluster API
 
 ### Prerequisites
 
-1. Install `kubectl` (see [here](http://kubernetes.io/docs/user-guide/prereqs/)).
+1. Install `kubectl` (see [here](http://kubernetes.io/docs/user-guide/prereqs/)). Because `kustomize` was included into `kubectl` and it's used by `cluster-api-provider-openstack` in generating yaml files, so version `1.14.0+` of `kubectl` is required, see [integrate kustomize into kubectl](https://github.com/kubernetes/enhancements/issues/633) for more info.
```

Comment:
Can you update the link to https://kubernetes.io/docs/tasks/tools/install-kubectl/